### PR TITLE
feat(AJV): Add Rest api logs definition

### PR DESCRIPTION
**Pull request by fredericbarthelet**
Addresses: #8018

### CODE REVIEW

**Review by medikoo — CHANGES_REQUESTED** [review]
It appears tests relied on `apiGatewayValidLogLevels` (see CI fail). If its' indeed needed over there, I would just hardcode it to test file

**Comment by fredericbarthelet** [comment]
> It appears tests relied on `apiGatewayValidLogLevels` (see CI fail). If its' indeed needed over there, I would just hardcode it to test file

Hardcoded it for the tests that needs it.

File: lib/plugins/aws/provider/awsProvider.js [hunk 1]
```diff
@@ -465,6 +465,18 @@ class AwsProvider {
...
+                restApi: {
+                  type: 'object',
+                  properties: {
+                    accessLogging: { type: 'boolean' },
+                    executionLogging: { type: 'boolean' },
+                    format: { type: 'string' },
+                    fullExecutionData: { type: 'boolean' },
+                    level: { enum: ['INFO', 'ERROR'] },
+                    role: { $ref: '#/definitions/awsArn' },
+                    roleManagedExternally: { type: 'boolean' },
```
Comment (medikoo) [inline lib/plugins/aws/provider/awsProvider.js:477]: It'll be nice to mark those properties as exclusive

**Review by medikoo — COMMENTED** [review]
Thank you @fredericbarthelet it looks great. I have just one minor improvement suggestion. Let me know

**Review by medikoo — APPROVED** [review]
Thank you @fredericbarthelet !

**Pull request merged**